hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -276,9 +277,8 @@ public void testInsertUpsertCluster(HoodieTableType tableType) throws Exception
   /**
    * Test rollback of various table operations sync to Metadata Table correctly.
    */
-  //@ParameterizedTest
-  //@EnumSource(HoodieTableType.class)
-  @Disabled
+  @ParameterizedTest
+  @EnumSource(HoodieTableType.class)
```

Comment:
this test along with testErrorCases was flaky in CI and to unblock CI disabled it sometime back. now with this fix, should not be flaky anymore.